org.lflang/src/org/lflang/generator/ts/TSConstructorGenerator.kt

```diff
@@ -106,6 +106,20 @@ class TSConstructorGenerator (
         return connectionInstantiations.joinToString("\n")
     }
 
+    fun generateFederateConfigurations(): String {
```

Comment:
Would you please add documentation for this function? It is a bit unclear what it does.

Reply:
Sure, added comments in https://github.com/lf-lang/lingua-franca/pull/538/commits/a64a9e9b9b9c1d56ed13a2ef01b42dce08c8e9c8.